backup/pvc/bin/backup.sh

```diff
@@ -22,8 +22,17 @@ done
 
 _log "INFO" "[backup] running backup ${BACKUP_NUMBER}"
 touch "${TRAP_FILE}"
+
+# since at this point we are sure that no other backup process is running, we can clean up stale tmp dirs
```

Comment:
Is it safe to assume that? I mean, trap file protects us against concurrent runs with the same number, but is it possible that another backup with a lower or higher number runs concurrently?